### PR TITLE
CSV/report changes

### PR DESCRIPTION
**Pull request by rameshtathe**
I have made the changes, Can you please review?

Made changes to update csv report fromat

### CODE REVIEW

File: python/avi/netscaler_converter/ns_util.py [hunk 1]
```diff
@@ -634,35 +641,42 @@ def policy_converter(policies, rule_index1, bind_patset, patset_config,
...
         if not is_policy_successful:
-            LOG.info('Datascript : %s' % bind_cmd)
-            add_status_row(bind_cmd, 'Datascript')
+            LOG.info('Datascript : %s %s' % (bind_cmd, b_cmd))
+            add_status_row(bind_cmd, b_cmd, full_cmd, STATUS_DATASCRIPT)
         else:
-            LOG.info('Successful: %s' % bind_cmd)
-            add_status_row(bind_cmd, 'Successful')
+            LOG.info('Successful: %s %s' % (bind_cmd, b_cmd))
+            add_status_row(bind_cmd, b_cmd, full_cmd, STATUS_SUCCESSFUL)
```
Comment (grastogi23) [inline python/avi/netscaler_converter/ns_util.py]: should there be an object passed down when status is successful?

File: python/avi/netscaler_converter/ns_util.py [hunk 2]
```diff
@@ -689,7 +707,8 @@ def get_rewrite_action(policy_name, policy_rules, rewrite_action_config):
             if matches:
                 value = {'val': matches[0]}
                 policy_rule['hdr_action'][0]['hdr']['value'].update(value)
-                LOG.info('Successful: %s' % cmd)
+                LOG.info('Successful: %s %s' % (cmd, policy_name))
+                add_status_row(cmd, policy_name, full_cmd, STATUS_SUCCESSFUL)
```
Comment (grastogi23) [inline python/avi/netscaler_converter/ns_util.py]: same here. shouldn't we pass the policy object / rule here?

**Comment by grastogi23** [comment]
can you pl. ensure all the conv_status with successful and partial have objects passed down.

File: python/avi/netscaler_converter/csvs_converter.py [hunk 3]
```diff
@@ -157,12 +158,10 @@ def convert(self, ns_config, avi_config, vs_state):
                                 policy = copy.deepcopy(policy)
                                 policy['targetLBVserver'] = targetVserver
                                 cs_vs_policies.append(policy)
-                                ns_util.add_status_row(pl_cmd, 'successful')
-                                LOG.info('Successful : %s' % pl_cmd)
+                                LOG.info('Conversion successful : %s' % pl_cmd)
```
Comment (grastogi23) [inline python/avi/netscaler_converter/csvs_converter.py:161]: there is no add_status_row here. It seems to have been previously present

File: python/avi/netscaler_converter/csvs_converter.py [hunk 4]
```diff
@@ -104,12 +106,11 @@ def convert(self, ns_config, avi_config, vs_state):
                 if clttimeout:
                     ns_util.add_clttimeout_for_http_profile(http_prof, avi_config, clttimeout)
                     clt_cmd = cmd + ' cltTimeout %s' % clttimeout
-                    ns_util.add_status_row(clt_cmd, 'Successful')
-                    LOG.info('Successful : %s' % clt_cmd)
+                    LOG.info('Conversion successful : %s' % clt_cmd)
```
Comment (grastogi23) [inline python/avi/netscaler_converter/csvs_converter.py:109]: I don't see replacement for add_status_row here.

**Review by grastogi23 — CHANGES_REQUESTED** [review]

File: python/avi/netscaler_converter/csvs_converter.py (hunk 4 above)
Comment (rameshtathe) [reply python/avi/netscaler_converter/csvs_converter.py:109]: @grastogi23 Here,  attribute clttimeout is successfully converted and added into the avi config object, though we add complete avi config object on completion of complete command conversion to CSV report some where below. So here it is not required to add_status_row.

File: python/avi/netscaler_converter/csvs_converter.py (hunk 3 above)
Comment (rameshtathe) [reply python/avi/netscaler_converter/csvs_converter.py:161]: @grastogi23 Exactly same case as above. Here we have successfully converted attribute and not complete command yet so not added to CSV report but we only update AVI config object here which is logged to CSV report on completion of complete command.

**Review by grastogi23 — APPROVED** [review]

**Pull request merged**